### PR TITLE
bridge: "Any" superuser bridge means just the first

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

**Comment by mvollmer** [comment]
This is a cleaner solution than the "explicitly-only" flag used earlier.

I kept the name "any" for the new semantics instead of changing to something else, such as "default" just to minimize protocol changes between components that are not always kept in synch.

File: src/bridge/cockpitrouter.c [hunk 1]
```diff
@@ -1713,73 +1711,43 @@ cockpit_router_dbus_startup (CockpitRouter *router)
...
-      if (rule->config)
-        cockpit_json_get_bool (rule->config, "only-explicitly", FALSE, &only_explicitly);
+  router->superuser_init_in_progress = TRUE;
 
-      if (rule_id && ((router->superuser_init_id == NULL && !only_explicitly)
-                      || (router->superuser_init_id != NULL && g_str_equal (router->superuser_init_id, rule_id))))
+  for (GList *l = router->rules; l; l = g_list_next (l))
+    {
+      gchar *rule_id = rule_superuser_id (l->data);
+      if (rule_id && (id == NULL || g_str_equal (id, rule_id)))
```
Comment (martinpitt) [inline src/bridge/cockpitrouter.c:1745]: so "any" means `id == NULL`, right? That seems fine. But to be on the safe side, should we then move the sudo rule in the manifest to be the first one, to be on the safe side?

Comment (mvollmer) [reply src/bridge/cockpitrouter.c:1745]: It is the first, no?

Comment (mvollmer) [reply src/bridge/cockpitrouter.c:1745]: > so "any" means `id == NULL`, right?

Yes, exactly.

Comment (martinpitt) [reply src/bridge/cockpitrouter.c:1745]: Argh, sorry -- I was looking at my test branch, which had it in the other order.

**Pull request merged**